### ISSUE TITLE
combining results of multi-traits using --no-split option

### ISSUE THREAD

**Issue by o147942** [rgcgithub/regenie#225]
Hi,
I used 10 traits in step 2.
I found strange results when I used the no-split option.
I think my data need an outer join because each trait had different missing values.
The null results were ignored, and there was no their column.
Therefore, I cannot distinguish the result of each trait.
Please check my results, 1st and 2nd row.

```
CHROM GENPOS ID ALLELE0 ALLELE1 A1FREQ INFO N TEST BETA.Y1 SE.Y1 CHISQ.Y1 LOG10P.Y1 BETA.Y2 SE.Y2 CHISQ.Y2 LOG10P.Y2 BETA.Y3 SE.Y3 CHISQ.Y3 LOG10P.Y3 BETA.Y4 SE.Y4 CHISQ.Y4 LOG10P.Y4 BETA.Y5 SE.Y5 CHISQ.Y5 LOG10P.Y5 BETA.Y6 SE.Y6 CHISQ.Y6 LOG10P.Y6 BETA.Y7 SE.Y7 CHISQ.Y7 LOG10P.Y7 BETA.Y8 SE.Y8 CHISQ.Y8 LOG10P.Y8 BETA.Y9 SE.Y9 CHISQ.Y9 LOG10P.Y9 BETA.Y10 SE.Y10 CHISQ.Y10 LOG10P.Y10 EXTRA
 0.886282 0.345194 6.59202 1.98954 -0.246272 0.148515 2.74973 1.01202 0.0557337 0.163997 0.115495 0.13432 -1.27157 4.80894 0.0699172 0.101573 -12.7762 7.31114 3.05373 1.09392 -0.177102 0.418849 0.178786 0.17236 -1.72625 2.42182 0.508068 0.322415 NA
 0.896622 0.346123 6.71056 2.01844 -0.24367 0.14891 2.67769 0.992409 0.060422 0.164433 0.135024 0.146741 -1.16206 4.82064 0.0581092 0.0917779 -12.8882 7.33677 3.08585 1.10251 -0.191817 0.419966 0.208615 0.188522 -1.47084 2.42702 0.367272 0.264006 NA
1 692794 1:692794:CA:C CA C 0.093775 0.340784 1180 ADD 0.580225 2.93064 0.0391983 0.0741434 0.0224412 0.163112 0.0189286 0.0503312 -0.0437677 0.0702526 0.388134 0.273043 0.0591931 0.0774521 0.584084 0.351917 5.44905 2.68519 4.11804 1.37235 6.13826 4.78326 1.64681 0.700287 0.0725352 0.197716 0.134591 0.146473 4.18508 3.3497 1.56097 0.674643 -1.83381 4.04243 0.20579 0.187028 0.403502 1.3635 0.0875751 0.115045 NA
1 711310 1:711310:G:A G A 0.052539 0.343864 1180 ADD -0.418777 3.80741 0.0120978 0.0398066 0.135616 0.212869 0.40588 0.280612 -0.0873634 0.0915926 0.909786 0.468302 0.0715318 0.101088 0.500723 0.319499 3.42279 3.30427 1.07302 0.522497 -5.72957 4.97594 1.32585 0.60285 -0.137473 0.258003 0.283914 0.226106 -5.80108 5.88249 0.972513 0.489382 -0.438146 1.6496 0.0705468 0.102075 NA
```

**Comment by joellembatchou** [comment]
Hi,

There is a bug in the no-split function which does not properly output variants that have NA p-values for some traits. This will be fixed in the next release.

Cheers,
Joelle

**Comment by joellembatchou** [comment]
This should be fixed in v3.0.
